packetbeat/protos/mysql/mysql.go

```diff
@@ -217,7 +256,8 @@ func mysqlMessageParser(s *mysqlStream) (bool, bool) {
 			if m.seq == 0 {
 				// starts Command Phase
 
-				if m.typ == mysqlCmdQuery {
+				if m.typ == mysqlCmdQuery || m.typ == mysqlCmdStmtPrepare ||
+					m.typ == mysqlCmdStmtExecute || m.typ == mysqlCmdStmtClose {
```

Comment:
in case list of supported packet types keeps increasing we should introduce a helper function like:

```
func isRequest(type) (request bool, supported bool) {}
```

Reply:
introduce a helper function will be better